acceptance/suites/tests/authorization/hup_reloads_crl.rb

```diff
@@ -0,0 +1,68 @@
+test_name '(SERVER-1380) CRL should reload after a server HUP' do
+
+server = master.puppet['certname']
+
+teardown do
+  # Leave agents with working certs. Destroy the revoked cert on the master;
+  # remove all traces on the agent; make the agent request a shiny new cert; 
+  # have the master sign it; then use it to make sure it works.
+  agents.each do |a|
+    if not_controller(a)
+      hostcert_file = a.puppet['hostcert']
+      reqdir = a.puppet['requestdir']
+      nodefqdn = a.hostname
+      request_file = File.join(reqdir, nodefqdn + ".pem")
+
+      # FIXME: this is a dirty nasty hack that Beaker should handle seamlessly
```

Comment:
ah, I see, it's a hack for "detroy the revoked cert on the master, remove all traces on the agent, make the agent request a shiny new cert, have the master sign it, then use it to make sure it works"... so I assume you want a helper that actually does this in Beaker? is there a ticket for that? will this ever actually get fixed?